app/src/components/Sidebar.tsx

```diff
@@ -72,15 +99,15 @@ interface ISidebarGenreChannel {
     icon: string;
 }
 
-export interface ISidebarProps extends IEnhancedComponentProps{
+export interface ISidebarProps extends IEnhancedComponentProps {
     className?: string;
     hasSearch?: boolean;
 }
 
 export interface ISidebarState extends IEnhancedComponentState {
-    searchValue?: string;
     genre: string;
     icon: string;
+    shownGenres: { genre: string, icon: string }[];
```

Comment:
there's already an interface declared above this, `ISidebarGenreChannel`. It's best to avoid inline interfaces when you can